zeroex/orderwatch/order_watcher.go

```diff
@@ -853,23 +817,23 @@ func (w *Watcher) Cleanup(ctx context.Context, lastUpdatedBuffer time.Duration)
 		orderHashToEvents[order.Hash] = []*zeroex.ContractEvent{}
 	}
 
-	latestBlock, err := w.meshDB.FindLatestMiniHeader()
+	latestBlock, err := w.getLatestBlock()
 	if err != nil {
 		return err
 	}
 	// This timeout of 30min is for limiting how long this call should block at the ETH RPC rate limiter
 	ctx, cancel := context.WithTimeout(ctx, 30*time.Minute)
 	defer cancel()
-	orderEvents, err := w.generateOrderEventsIfChanged(ctx, ordersColTxn, orderHashToDBOrder, orderHashToEvents, latestBlock.Number, latestBlock.Timestamp)
+	orderEvents, err := w.generateOrderEventsIfChanged(ctx, orderHashToDBOrder, orderHashToEvents, latestBlock.Number, latestBlock.Timestamp)
 	if err != nil {
 		return err
 	}
 
-	if err := ordersColTxn.Commit(); err != nil {
-		logger.WithFields(logger.Fields{
-			"error": err.Error(),
-		}).Error("Failed to commit orders collection transaction")
-	}
+	// if err := ordersColTxn.Commit(); err != nil {
+	// 	logger.WithFields(logger.Fields{
+	// 		"error": err.Error(),
+	// 	}).Error("Failed to commit orders collection transaction")
+	// }
```

Comment:
Why is this commented out?

Reply:
We don't use transactions inside of order watcher anymore for reasons that I mentioned in the PR description. I just forgot to remove this code. It's removed now.